src/examples/TTSExamplePage.tsx

```diff
@@ -0,0 +1,90 @@
+'use strict';
+import { Button, FlatList, GestureResponderEvent, StyleSheet, Text, TextInput, View } from 'react-native';
+import React from 'react';
+import Tts, { Voice } from 'react-native-tts';
+import { Example } from '../components/Example';
+import { Page } from '../components/Page';
+
+export const TTSExamplePage: React.FunctionComponent<{}> = () => {
+  const exampleJsx = `import React from 'react';
+import {Button} from 'react-native';
+import Tts from 'react-native-tts';
+  
+function Example() {
+  const onSpeak = () => { Tts.speak("Hello World"); }
+
+  return (
+    <Button onPress={onSpeak} title="Speak"/>
+  );
+}
+  `;
+
+  const [text, setText] = React.useState('Hello World');
+
+  const [selectedVoice, setSelectedVoice] = React.useState<Voice>()
+  const [voices, setVoices] = React.useState<Voice[]>([]);
+
+  const onSpeak = () => { Tts.setDefaultVoice(selectedVoice!.id); Tts.speak(text); }
+
+  const onVoicePress = (evt: GestureResponderEvent, voice: Voice) => {
+    setSelectedVoice(voice)
+  }
+
+  const renderVoiceItem = ({ item }: any) => {
+    const voice = item as Voice;
+    return (
+      <View>
+        <Button
+          onPress={(evt) => onVoicePress(evt, voice)}
+          title={voice.name}
+          color={voice.id == selectedVoice?.id ? "#228B22" : "#841584"}
```

Comment:
nit: Could you change the colors here to #E6E6E6, 'red', or 'steelblue' so gallery colors are consistent across pages?